### ISSUE TITLE
Add "teams" feature to enable bragging on teams more easily

### ISSUE THREAD

**Issue by docrinehart** [HeadspringLabs/hsbot#74]
hsbot would store a collection of key/values linking users to teams
A sample set of commands might look like:
`
hsbot team create/delete <teamname>
hsbot team <teamname> add/remove <username>
hsbot brag on @<teamname> // would brag on each member of the team
`

**Comment by bardoloi** [comment]
@docrinehart good idea. question: would the benefit be worth the cost of maintaining the team's list of users (given how often people roll on/off projects)?

**Comment by docrinehart** [comment]
@bardoloi  I think for larger, longer-standing teams it would be more beneficial. Maybe instead of backing it with a collection in hsbot, it could be integrated with some other system that tracks people on projects. PM or something...

**Comment by erinknight242** [comment]
Maybe there could be a confirmation step, where hsbot lists back the members on the team so you know who you're about to send the brag for. If the list isn't up to date, you could cancel, edit the team, and re-brag
